kazu/training/modelling_utils.py

```diff
@@ -64,6 +79,51 @@ def get_label_list_from_model(model_config_path: PathLike) -> list[str]:
     return label_list
 
 
+class LSManagerViewWrapper:
+    def __init__(self, view: LabelStudioAnnotationView, ls_manager: LabelStudioManager):
+        self.ls_manager = ls_manager
+        self.view = view
+
+    def get_gold_ents_for_side_by_side_view(self, docs: list[Document]) -> list[list[Document]]:
+        result = []
+        for doc in docs:
+            doc_cp = copy.deepcopy(doc)
+            if PROCESSING_EXCEPTION in doc_cp.metadata:
+                logger.error(doc.metadata[PROCESSING_EXCEPTION])
+                break
+            for section in doc_cp.sections:
+                gold_ents = []
+                for ent in section.metadata.get("gold_entities", []):
+                    if isinstance(ent, dict):
```

Comment:
Just out of interested (as I don't have a comprehensive picture of the whole code base) - why is this necessary resp. why we use different formats (object vs. Dicts) when storing the gold entities?